pom.xml

```diff
@@ -121,7 +121,7 @@
         <rxjava.version>2.2.21</rxjava.version>
         <commons-codec.verion>1.15</commons-codec.verion>
         <jmh.version>1.37</jmh.version>
-        <awscrt.version>0.29.23</awscrt.version>
+        <awscrt.version>0.29.25</awscrt.version>
```

Comment:
Are there any runtime checks we need to update, to ensure the user has installed`>= 0.29.25`?